react/features/base/config/configWhitelist.js

```diff
@@ -123,6 +123,7 @@ export default [
     'liveStreamingEnabled',
     'localRecording',
     'minParticipants',
+    'moderatedRoomServiceUrl',
```

Comment:
I don't think it's a good idea to white-list it. Someone could use the external API to send you to some evil website.